draft-ounsworth-pq-composite-sigs.mkd

```diff
@@ -239,7 +248,7 @@ Each element of a CompositePublicKey is a SubjectPublicKeyInfo object one of the
 
 ~~~ Begin EDNOTE ~~~
 
-EDNOTE: there has been a fair amout of discussion among the authors about whether the component public key should contain a full SubjectPublicKeyInfo for each component algorithm, or whether the {algID, and algParams} should be move to the params of the PUBLIC-KEY or OID, and only the BIT STRINGs of the component public key values contained in the CompositePublicKey.
+EDNOTE: there has been a fair amount of discussion among the authors about whether the component public key should contain a full SubjectPublicKeyInfo for each component algorithm, or whether the {algID, and algParams} should be move to the params of the PUBLIC-KEY or OID, and only the BIT STRINGs of the component public key values contained in the CompositePublicKey.
```

Comment:
I wonder if it makes sense to keep this EDNOTE. Correct me if I'm wrong - it seems to me that already decided and (I think) discussion can be found in the github.

Reply:
Agreed. I've started a discussion on #81 and #83.